src/ripple/app/main/BasicApp.cpp

```diff
@@ -24,20 +24,22 @@ BasicApp::BasicApp(std::size_t numberOfThreads)
 {
     work_.emplace (io_service_);
     threads_.reserve(numberOfThreads);
+
     while(numberOfThreads--)
-        threads_.emplace_back(
-            [this, numberOfThreads]()
+    {
+        threads_.emplace_back([this, numberOfThreads]()
             {
-                beast::setCurrentThreadName(
-                    std::string("io_service #") +
-                        std::to_string(numberOfThreads));
-                this->io_service_.run();
+              beast::setCurrentThreadName("io svc #" +
+                  std::to_string(numberOfThreads));
+              this->io_service_.run();
             });
+    }
```

Comment:
Nit: Threads are named in reverse order to how they are emplaced and run. I am okay as is but it could be to:
```
    for(std::size_t i = 1; i <= numberOfThreads; ++i)
    {
        threads_.emplace_back([this, i]()
            {
              beast::setCurrentThreadName("io svc #" + std::to_string(i));
              this->io_service_.run();
            });
    }
```